File: 2073-time-needed-to-buy-tickets/2073-time-needed-to-buy-tickets.java
class Solution {
    public int timeRequiredToBuy(int[] tickets, int k) {
        int ans = 0;
        for(int i = 0; i<tickets.length; i++) {
            int val = i > k ? tickets[k] - 1 : tickets[k];
            ans += Math.min(tickets[i], val);
        }

        return ans;
    }
}

File: 2073-time-needed-to-buy-tickets/2073-time-needed-to-buy-tickets.js
/**
 * @param {number[]} tickets
 * @param {number} k
 * @return {number}
 */
var timeRequiredToBuy = function(tickets, k) {
    let ans = 0, n = tickets.length;
    for(let i = 0; i<n; i++) {
        let val = i > k ? tickets[k] - 1 : tickets[k];
        ans += Math.min(val, tickets[i]);
    }

    return ans;
};